ci-operator/step-registry/janus-idp/backstage-showcase/janus-idp-backstage-showcase-commands.sh

```diff
@@ -9,7 +9,7 @@ GIT_PR_NUMBER=$(echo "${JOB_SPEC}" | jq -r '.refs.pulls[0].number')
 echo "GIT_PR_NUMBER : $GIT_PR_NUMBER"
 GITHUB_ORG_NAME="janus-idp"
 GITHUB_REPOSITORY_NAME="backstage-showcase"
-NAME_SPACE=showcase-ci
+NAME_SPACE=showcase
```

Comment:
@subhashkhileri do we need to set this variable? it would be overridden from 'openshift-ci-tests.sh' call anyways.

Reply:
true it would be overridden. it was there so I just kept it as it for reference and just matched the value `showcase`

Reply:
I will remove it.